### DISCUSSION TITLE
Error message in the console regarding img src not able to load

### DISCUSSION THREAD

**Discussion by NirachaMarchett** [neuefische/questions#743] [Web] [answered]
Hi everyone,

I've been trying to make the image appears but keep getting the error message in the console. I double check path to the image file and it seems to be correct so I am not sure why the image still broken (see below). Could someone please help me fix this issue? 

here is the error message;
![Capture](https://github.com/neuefische/questions/assets/135506311/3680e7ad-3942-4e91-b9ff-00cd80baee1c)

and this is the codes:

```
import React from 'react';
import Head from 'next/head'
import { Inter } from 'next/font/google'
import styles from '@/styles/Home.module.css'
import Link from 'next/link'
import GameIcon from '../../public/assets/game-icon.png';

const inter = Inter({ subsets: ['latin'] })

export default function Home() {
  return (
    <>
      <Head>
        <title>Home page</title>
      </Head>
      <main className={`${styles.main} ${inter.className}`}>
        <div className={styles.welcomeText}>
          <p>
            Welcome to
          </p>
          <h2>
            PairLingo
          </h2>
        </div>
        <img src={GameIcon} alt='Home screen image' style={{ width: '200px', height: '200px', marginTop: '100px' }}/>
            <Link href="/player" className={styles.customLink}>START</Link>
      </main>
    </>
  )
}
```

here is the repository of the project
[project](https://github.com/NirachaMarchett/couple-vocabs-game)

Thank you so much in advance.

**Comment by moonwave99** [comment, marked as answer]
Hey Niracha, good to se you again!

Since the img is already in the `public` folder,  you can reference it directly as:

```js
<img src="/assets/game-icon.png"/>
```

Even better, use the `Image` component from `next/image`.

Let us know if it works ^^

**Reply by NirachaMarchett** [reply]
Hi Diego, It works! thank you soo much! :)
